README.md

```diff
@@ -1,20 +1,88 @@
 # Flow Data Provisioning Service
 
+The Flow Data Provisioning Service aims at providing a scalable and efficient way to access the execution state of past
```

Comment:
Would read a bit better if you made `past sporks` explicit.

Reply:
So you mean to have `access the execution state of past sporks and live sporks` instead of `access the execution state of past and live sporks` right? It sounds weird to me to repeat sporks twice in that context.

Reply:
`past and live sporks` sounds strange, because there is only one spork. Probably a clearer way to express this would be `to access the history of the Flow execution state, both for the current live sporks and for past sporks.`